templates/fi/contact.html

```diff
@@ -17,7 +17,7 @@ <h2>Ota Yhteyttä</h2>
             <label for="message">Viestisi</label>
             <textarea id="message" name="message" rows="6" required></textarea>
             
-            <button type="submit" class="cta-button">Lähetä</button>
+            <button type="submit">Lähetä</button>
```

Comment:
Cta button is required, for it to work.
```suggestion
            <button type="submit" class="cta-button">Lähetä</button>
```